cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -0,0 +1,88 @@
+{% extends "_layouts/oah-base.html" %}
+{% import "molecules/social-media.html" as social_media with context %}
+{% set active_page = "closing-disclosure" %}
+{% set page_js = "closing-disclosure" %}
+
+{% block title %}Closing Disclosure Explainer{% endblock title %}
+{% block metadescription %}Use the CFPB's tool to double-check that all the details about your loan are correct on your Closing Disclosure.{% endblock metadescription %}
+
+{% block content %}
+<main class="content">
+  <div class="content_wrapper">
+      <nav class="breadcrumbs" aria-label="Breadcrumbs">
+        <a href="/owning-a-home/" class="breadcrumbs_link">Owning a Home</a>
+      </nav>
+  </div>
+  <div class="content_wrapper">
+    <div class="form-explainer content_main {% if subnav %}col-9{% endif %} u-pb0">
```

Comment:
I'm going to leave this as-is, because they will need to get refactored once it goes to Wagtail.